src/main.cpp

```diff
@@ -58,6 +60,13 @@ int main(int argc, char* argv[])
 #endif
     signal(SIGINT, xcpp::stop_handler);
 
+#ifdef XEUS_CPP_USE_DEBUGGER
+    nl::json debugger_config;
+    debugger_config["lldb"]["initCommands"] = {
+        "settings set plugin.jit-loader.gdb.enable on"
+    };
+#endif
```

Comment:
This does not look correct. The use of `ifdef`. Let's discuss it in the meeting.
We need to do this based on the metadata of the `kernel.json`.